probe-rs/src/architecture/arm/component/itm.rs

```diff
@@ -135,4 +242,22 @@ mod register {
             Self(0x0000_0000)
         }
     }
+
+    memory_mapped_bitfield_register! {
```

Comment:
Can we use this new nice type in `tx_enable` too? :)

Reply:
Done! :)